spec/models/taxon_merge_spec.rb

```diff
@@ -1,10 +1,10 @@
 require File.dirname(__FILE__) + '/../spec_helper.rb'
 
 def setup_taxon_merge
-  input_genus = Taxon.make!( rank: Taxon::GENUS )
-  @input_taxon1 = Taxon.make!( rank: Taxon::SPECIES, parent: input_genus )
-  @input_taxon2 = Taxon.make!( rank: Taxon::SPECIES, parent: input_genus )
-  @input_taxon3 = Taxon.make!( rank: Taxon::SPECIES, parent: input_genus )
+  @input_genus = Taxon.make!( rank: Taxon::GENUS )
```

Comment:
If this isn't always going to be a genus and will be assigned a new rank in different tests, please call it `@input_ancestor` or something